tests/CMakeLists.txt

```
@@ -6,31 +6,55 @@ function(add_vue_test TestName)
   )
 endfunction()
 
-function(add_cypress_test TestName)
+function(cypress_shared)
   if(DEFINED ENV{APP_URL})
-    set(APP_URL $ENV{APP_URL})
+    set(APP_URL $ENV{APP_URL} PARENT_SCOPE)
```

It's safer to use quotes here in case `$ENV{APP_URL}` contains a space, because if does, `APP_URL` will be set to a cmake list (semicolon-separated values) instead of a single string.
```suggestion
    set(APP_URL "$ENV{APP_URL}" PARENT_SCOPE)
```

---

There probably worse problems if someone is trying to develop with an APP_URL which contains a space, but I'll make the change for thoroughness...